datatypes/papas/DefinitionsCollections.h

```diff
@@ -0,0 +1,42 @@
+//
+//  DefinitionsCollections.h
+//  papas
+//
+//  Created by Alice Robson on 26/05/16.
+//
+//
+
+#ifndef DefinitionsCollections_h
+#define DefinitionsCollections_h
+
+#include "Id.h"
+//#include <map>
+#include <unordered_map>
+#include <vector>
+namespace papas {
+class Cluster;
```

Comment:
I think this is infact the fcc coding standard. See:
https://github.com/jlingema/FCCSW/blob/master/doc/CppCodingStyleGuidelines.md#indentation
All the code has been passed through a clang formatter with the fcc clang file.